src/renderer/chat/ChatTab.ts

```diff
@@ -139,6 +139,7 @@ export default class ChatTab {
     userElt.appendChild(nicknameElt);
 
     this.usersTreeView.append(userElt, "item");
+    if(!this.hasUser(name)) this.users.push(name);

```

Comment:
Missing space between "if" and "(". Also this only sort of fixed user management. I fixed it fully in b7270e85e018b5030d174d84252b780c024166a8.